src/Modules/Engine.cpp

```diff
@@ -56,6 +56,10 @@ Variable sar_tick_debug("sar_tick_debug", "0", 0, 3, "Output debugging informati
 
 Variable sar_cm_rightwarp("sar_cm_rightwarp", "0", "Fix CM wrongwarp.\n");
 
+float g_cur_fps = 0.0f;
```

Comment:
since we're now in `Modules/Engine.cpp` anyway, this would probably be nicer as a field on `Engine`.